index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+function plugin(onEnd) {
```

Comment:
Name the plugin more specific for stack traces? Something like: `superAgentTimerPlugin`?